core/src/executor/evaluate/function.rs

```diff
@@ -215,6 +215,29 @@ pub fn substr(
 
 // --- float ---
 
+pub fn abs(name: String, n: Evaluated<'_>) -> Result<Value> {
+    match n.try_into()? {
+        Value::I8(v) => Ok(Value::I8(v.abs())),
+        Value::I64(v) => Ok(Value::I64(v.abs())),
+        Value::Decimal(v) => Ok(Value::Decimal(v.abs())),
+        Value::F64(v) => Ok(Value::F64(v.abs())),
+        Value::Null => {
+            return Ok(Value::Null);
+        }
+        _ => {
+            return Err(EvaluateError::FunctionRequiresFloatValue(name).into());
+        }
+    }
```

Comment:
```suggestion
        Value::Null => Ok(Value::Null),
        _ => Err(EvaluateError::FunctionRequiresFloatValue(name).into())
    }
```
The rest of the value passed in the `match` is used as the return value, so I think you can delete `return` too